apps/utils/scripts/run-bfd-server

```diff
@@ -248,9 +253,20 @@ setvar paths/files/truststore "$bin_dir/ssl/${truststore_file}"
 setvar paths/files/keystore "$bin_dir/ssl/${keystore_file}"
 setvar paths/files/war "$bin_dir/$war_file_name"
 
-setvar db/url "jdbc:postgresql://${db_host}:${db_port}/bfd?logServerErrorDetail=false"
-setvar db/username "bfd"
-setvar db/password "InsecureLocalDev"
+# Conditional database URL setting
+if [[ $use_test_db = "true" ]] ; then
+  # Use the test database URL
+  echo "Using test database: $test_db_url"
+  setvar db/url "jdbc:postgresql://bfd-test-aurora-cluster.cluster-clyryngdhnko.us-east-1.rds.amazonaws.com:5432/fhirdb?logServerErrorDetail=false"
+  setvar db/username "FOPH"
+  setvar db/password "bfd-test-aurora-cluster.cluster-clyryngdhnko.us-east-1.rds.amazonaws.com:5432/?Action=connect&DBUser=FOPH&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAYM3RJQIP76C4GXT7%2F20240819%2Fus-east-1%2Frds-db%2Faws4_request&X-Amz-Date=20240819T132059Z&X-Amz-Expires=900&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjED4aCXVzLWVhc3QtMSJIMEYCIQDhYVuiLdF96RorOMFXPvTBMClM1xcyDc1PTWrvq0cC1wIhAPbvRsOLVlLDOfkBkJbVhpLZxhRpnqyAhvzsRIdKsdreKu4BCEYQBBoMNTc3MzczODMxNzExIgxZVIRWT%2BoMNODa7BQqywHjkM8DvZqonJVEIhUPnbuqBm9DdY9dA52yFsUppkk7nBgqG5e2mip%2Bl7Kb8ixQfAr5XgsNIV5dLp9PEaUNJQJPwKe4FgClpo1yVGJvJx%2FhafyRfm8wRrN91kLRu2V3XVjgLBfX54aob1PoSpSn62uuYeEoQVvLD1U8kVYwVN6VDQa6jPLE%2FLphpppiooHcfmp7r41M7lOiNGSDiQHHMmspM9AWnUgaZpdDXNcNuIvlC9b6q8ON3JT75iND%2BPTExyKyfrIQK2xMAhdTCjC1jY22BjqXAQ94Dzm%2FgCkj9DyVlL4IpSQbniALi6l9cXH4plnNGAGgtxJSlQXGctpsZ%2FPBfx%2BQ%2B3qZtc07PzLN%2Fdr5R0xXSeDUkeYBcdbM%2FcMqt1%2FFdgTv2WCLj0VHkBPxvp%2FzR822wAtI30eLkASzHoKZ2sVv5iRt0CrxtvaOI9S00Tk53Ci%2ByWCKCBbIjNTUm145l667E68nizHIslA%3D&X-Amz-Signature=ab647b90574e9d89bdc2a2e84b2f8acef275ffc5df4daa92dfd6e846465bcc50"
+else
+  # Use the local database configuration
+  echo "Using local database: $db_host:$db_port"
+  setvar db/url "jdbc:postgresql://${db_host}:${db_port}/bfd?logServerErrorDetail=false"
+  setvar db/username "bfd"
+  setvar db/password "InsecureLocalDev"
+fi
```

Comment:
Yes, you will need to include a separate variable, so that the user can set their own user-name and current password. Please delete these from your PR; they will probably generate a security alert.